mobile/envoy_build_config/extension_registry.cc

```diff
@@ -81,6 +82,7 @@ void ExtensionRegistry::registerFactories() {
   Envoy::Network::forceRegisterGetAddrInfoDnsResolverFactory();
   Envoy::Extensions::RequestId::forceRegisterUUIDRequestIDExtensionFactory();
   Envoy::Server::forceRegisterDefaultListenerManagerFactoryImpl();
+  Envoy::Server::forceRegisterApiListenerManagerFactoryImpl();
```

Comment:
should we force register in here only if `DUSE_API_LISTENER` is set?

Reply:
I was going to do that but I think it requires a bazel flag and additional plumbing to be useful given the library include